conf/application.conf

```diff
@@ -57,10 +57,6 @@ urBanner{
    link = "https://signup.take-part-in-research.service.gov.uk/?utm_campaign=PBIK_FPOS&utm_source=Survey_Banner&utm_medium=other&t=HMRC&id=67"
 }
 
-auditing {
```

Comment:
this can be reverted, as datastream keep having local problems and we dont really need auditing when run application locally, rely on default set to false in bootstrap library